static/js/1989.js

```diff
@@ -49,12 +50,14 @@ let doSubmit = function () {
     body: data,
   })
   .then((response) => response.json())
-  .then((json) => {
+  .then((json) => {r
```

Comment:
This results in an error in the frontend when running this JavaScript code.

Reply:
Just fixed it! sorry for making a stupid mistake

- Once all the mosaics have been generated, the homepage will feature party confetti effects.
- If the user doesn't upload the base image or makes another bad request, the page will feature sad face confetti effects.